README.md

```diff
@@ -16,11 +16,12 @@ The following configuration runs a DHCPv6-only server, listening on all the inte
 
 ```
 server6:
-    listen: '[::]:547'
+    listen: '[::]:547'       # this listens on every interface
+    # - "[fe80::1%eth0]:547" # this will listen on eth0
```

Comment:
Not sure if this makes it clear that `listen:` is normally a list, but also accepts a single address as a string if there is only one. Maybe the simplest is to only document the list version